assets/src/edit-story/app/story/actions/useSavePost.js

```diff
@@ -36,7 +66,8 @@ function useSavePost( {
 	const savePost = useCallback( () => {
 		if ( ! isSaving ) {
 			setIsSaving( true );
-			saveStoryById( storyId, title, status, pages, postAuthor, slug ).then( ( post ) => {
+			const content = getStoryMarkupFromPages( pages );
```

Comment:
Love how simple this is! :+1:

Reply:
I have a spinner for saving, so this takes some time then the spinner will show. Should `getStoryMarkupFromPages` return a promise?

Reply:
Currently it's just rendering so shouldn't take any time.